website/static/img/databricks_tutorial/images/packages.yml

```diff
@@ -0,0 +1,3 @@
+packages:
```

Comment:
@amychen1776 Is this packages.yml also OK to remove?

Reply:
removed!